easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.1-intel-2015a-eb-deps.eb

```diff
@@ -0,0 +1,51 @@
+name = 'OpenFOAM'
+version = '2.3.1'
+versionsuffix = '-eb-deps'
```

Comment:
No idea, we have discussed about this before, and it was not better solution. It could be `-externaldeps` or `-externaldependencies` but thats seem to be quite long.